cmd/daprd/components/wfbackend.go

```diff
@@ -0,0 +1,24 @@
+//go:build allcomponents
+
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package components
+
+import (
+	wfbeLoader "github.com/dapr/dapr/pkg/components/wfbackend"
+)
+
+func init() {
+	wfbeLoader.DefaultRegistry.RegisterComponent(wfbeLoader.NewWorkflowBackendComp, "actor", "sqlite")
```

Comment:
The goal of having these files is that it makes it so components can be enabled or disabled simply by creating/deleting files.

We need to have a separate file for each workflow backend component, so they can be enabled/disabled at will.

This is related to my earlier point around the fact that we need to have more decoupling so we can do things more "drop-in"...

Reply:
I initially I had created separate files for each backend component, but I merged them together because there was no other change except the name of the component.

Now also, if we have to add or remove a backend, we only need to remove or add the new component name. In both cases we need to make changes and release it. Do you see other benefits of having these components in their own file?